code/modules/paperwork/records/info/zayin.dm

```diff
@@ -99,8 +99,9 @@
 	Max PE Boxes : 10	<br>
 	Qliphoth Counter : X	<br>
 	Work Damage Type : Red	<br>
-	Work Damage : Low	<br>
-	- Once a work is done on we can change anything, the agent working will get hurt in exchange for a temporary buff.	<br>
+	Work Damage : See Below	<br>
+	- No matter if a worktick fails or succeeds, the employee working on T-09-85 took damage.	<br>
```

Comment:
Just keep work damage low and add note below saying that it deals work damage continuously.
"Worktick" doesn't sound very fitting for something that is meant to be used in-character.